### PR TITLE
dev-util/gammaray: new package

### PR DESCRIPTION
**Pull request by Petross404**
High-level runtime introspection tool for Qt applications

Package-Manager: Portage-2.3.36, Repoman-2.3.9

### CODE REVIEW

**Comment by Petross404** [comment]
The [previous](https://github.com/gentoo/gentoo/pull/7084) one.

File: dev-util/gammaray/gammaray-9999.ebuild [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/KDAB/GammaRay.git"
+else
+	SRC_URI="https://github.com/KDAB/GammaRay/releases/download/v${PV}/${P}.tar.gz"
+	KEYWORDS="~amd64"
+fi
+
+LICENSE="GPL-2 BSD MIT"
+SLOT=0
+IUSE="3d bluetooth concurrent designer -doc geolocation printsupport
```
[the remark this answers is not in the document]
Comment (Petross404) [reply dev-util/gammaray/gammaray-9999.ebuild]: No specific reason actually.

[the remark this answers is not in the document]
Comment (a17r) [reply dev-util/gammaray/gammaray-9999.ebuild]: are you going to fix it?

[the remark this answers is not in the document]
Comment (Petross404) [reply dev-util/gammaray/gammaray-9999.ebuild]: Fixed

**Pull request closed without merging**